example.ini

```diff
@@ -0,0 +1,21 @@
+[app:main]
+factory = reddit_service_ads_tracking:make_wsgi_app
+
+metrics.namespace = reddit_service_ads_tracking
```

Comment:
Do you think you guys are gonna have more services along these lines? Maybe having your own namespace makes sense?

    metrics.namespace = ads.tracking.production

?